src/views/quiz-settings/quiz-settings.js

```diff
@@ -0,0 +1,146 @@
+class QuizSettings {
+    quizAbout;
+    questionsNum;
+    questionsType;
+
+    static createRadioButton(value, settingName) {
+        const div = document.createElement('div');
+        div.setAttribute('class', 'radioButton')
+        const button = document.createElement('input');
+        button.setAttribute('id', value);
+        button.type = 'radio';
+        button.value = value;
+        button.name = settingName;
+        const label = document.createElement('label');
+        label.setAttribute('for', value);
+        label.innerText = value;
+        this.addEventListenerToComponent(button, 'click', settingName, value);
+        div.append(button, label);
+        return div;
+    }
+
+
+
+    static addEventListenerToComponent(component, event, settingName, value) {
+        component.addEventListener(event, () => {
+            if (settingName === 'quizAbout') {
+                this.quizAbout = value;
+            }
+            if (settingName === 'questionsType') {
+                this.questionsType = value;
+            }
+            if (settingName === 'questionsNum') {
+                this.questionsNum = component.value;
+            }
+            console.log(this.quizAbout, this.questionsType, this.questionsNum);
+        });
```

Comment:
Keep in mind, that
we can pass a function to the addEventListener, so 
```
() => {
            if (settingName === 'quizAbout') {
                this.quizAbout = value;
            }
            if (settingName === 'questionsType') {
                this.questionsType = value;
            }
            if (settingName === 'questionsNum') {
                this.questionsNum = component.value;
            }
            console.log(this.quizAbout, this.questionsType, this.questionsNum);
        }
```
could've been extracted